main.go

```diff
@@ -125,19 +155,22 @@ func argsParse(argv []string) (noPublish bool, sourcesArg []string, whiteListArg
 		fmt.Sprintf("%s %s", ProgramName, version), false)
 
 	// Parse argument values as usable types.
-	noPublish = arguments["--no-publish"].(bool)
-	sourcesArg = strings.Split(arguments["--sources"].(string), ",")
-	whiteListArg = arguments["--label-whitelist"].(string)
+	args.noPublish = arguments["--no-publish"].(bool)
+	args.sources = strings.Split(arguments["--sources"].(string), ",")
+	args.labelWhiteList = arguments["--label-whitelist"].(string)
+	args.oneshot = arguments["--oneshot"].(bool)
+	args.sleepInterval, _ = strconv.Atoi(arguments["--sleep-interval"].(string))
 
-	return noPublish, sourcesArg, whiteListArg
+	return args
 }
 
 // configureParameters returns all the variables required to perform feature
 // discovery based on command line arguments.
-func configureParameters(sourcesArg []string, whiteListArg string) (sources []source.FeatureSource, labelWhiteList *regexp.Regexp, err error) {
-	enabledSources := map[string]struct{}{}
-	for _, s := range sourcesArg {
-		enabledSources[strings.TrimSpace(s)] = struct{}{}
+func configureParameters(sourcesWhiteList []string, labelWhiteListStr string) (enabledSources []source.FeatureSource, labelWhiteList *regexp.Regexp, err error) {
```

Comment:
Are these changes to `configureParameters()` required?

Reply:
I tried to make the variable naming a bit more comprehensible/self-explanatory. The current naming confused me when reading the code for the first time.